### DISCUSSION TITLE
Are webrcade games downloaded locally?

### DISCUSSION THREAD

**Discussion by PaulLuigi** [webrcade/webrcade#52] [Q&A] [answered]
I assume that before playing a game, it's downloaded to a local file storage location somewhere on the device?  If so, is the local copy ever cleared?  

All of the games for the current systems are quite small, but I could imagine PS1 games (if they were ever added) being large enough that this is still possible but slower, and might require some support for limiting the local games cache size or clearing it.  Just curious!

**Comment by raz0red** [comment, marked as answer]
They are downloaded and reside in memory until the game is exited. In the future with larger games (CDs, etc.) it is highly likely that local storage will be utilized. Great question!
